WCMST1.sql_schema

```diff
@@ -154,7 +154,7 @@ CREATE TABLE IF NOT EXISTS `users` (
 ) ENGINE=InnoDB  DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci AUTO_INCREMENT=10 ;
 
 
-
+INSERT INTO `users` (`id`, `username`, `password`, `last_logged`) VALUES (NULL, 'admin', MD5('P@ssw0rd'), current_timestamp());
```

Comment:
```
CREATE TABLE IF NOT EXISTS `users` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `username` varchar(30) COLLATE utf8_unicode_ci NOT NULL,
  `password` varchar(50) COLLATE utf8_unicode_ci NOT NULL,
  `last_logged` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
  PRIMARY KEY (`id`),
  UNIQUE KEY `username` (`username`)
) ENGINE=InnoDB  DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci AUTO_INCREMENT=10 ;

```
@vishnudas-raveendran @luckymurari If I understand correctly, `id` column here has a `NOT NULL` constrain and we would get an error.

Reply:
I think we could skip the `id` and `last_logged ` columns as they will be auto-generated/default.

Reply:
Yes true. we must skip that. Oversight on my part, sorry. Reverted pull request #13